### DISCUSSION TITLE
Broken development environment variables?

### DISCUSSION THREAD

**Discussion by bscaspar** [vercel/vercel#5456] [Help] [answered]
I just updated the Vercel CLI to the latest and ran `vc env pull` to grab my updated development environment variables, many of which have existed for a few months now. 

The values that this command pulled down appear to be secret keys instead of the string values they used to be. They all start with `sec_`. This would be ok if they were somehow decrypted when I run `next dev` but that's not the case (not surprisingly since the dev env vars can't be encrypted according to the [docs](https://vercel.com/docs/platform/environment-variables)).

I understand that env vars have gone through some updates lately and that Preview and Production env vars are encrypted, but I am confused as to why this would happen for the development environment variables. Thinking it's a bug but want to see if other folks have encountered something similar before opening an issue.

Has anyone else encountered this? Anything I might be doing wrong?

**Comment by mcsdevv** [comment]
Hello @bscaspar, sorry about this, we'll get this fixed shortly.

For now, you can use the 19.2.0 CLI version to workaround the issue.

**Comment by bscaspar** [comment]
Thanks for confirming!

**Comment by mcsdevv** [comment, marked as answer]
Hey @bscaspar, we've released version [21.0.1](https://github.com/vercel/vercel/releases/tag/vercel%4021.0.1) with the fix.

**Reply by bscaspar** [reply]
Just confirmed, it's working as expected now, thank you for the quick turn around!
